src/Components/OpenModelControl.jsx

```diff
@@ -20,9 +25,27 @@ import {handleBeforeUnload} from '../utils/event'
  */
 export default function OpenModelControl({fileOpen}) {
   const [isDialogDisplayed, setIsDialogDisplayed] = useState(false)
+  const [orgNamesArr, setOrgNamesArray] = useState(['loading'])
+  const {user} = useAuth0()
   const theme = useTheme()
-
-
+  const accessToken = useStore((state) => state.accessToken)
+  useEffect(() => {
+    /**
+     * Asynchronously fetch organizations
+     *
+     * @return {Array} organizations
+     */
+    async function fetchOrganizations() {
+      const orgs = await getOrganizations(accessToken)
+      const orgNamesFetched = Object.keys(orgs).map((key) => orgs[key].login)
```

Comment:
@oo-bldrs perhaps I misunderstood... I tried to replace org with @org per GitHub convention and VS code was complaining about it.

Reply:
Per discussion, I think Ogali is suggesting the '@' is part of the display string for users and orgs, as that's common in git, as you can see when you @OlegMoshkovich someone.

I'd say that's more of a display format tho, since we often pass org and usernames without it.  So I'd keep the bare org and user names here, and then when you go to display them below, do smth like `` `@${orgName}` ``